src/Hintman/Suggestion/TrailingSpaces.hs

```diff
@@ -0,0 +1,34 @@
+{-# LANGUAGE NamedFieldPuns #-}
+
+module Hintman.Suggestion.TrailingSpaces
+       ( getTrailLines
+       , toLines
+       , suggest
+       , suggestRaw
+       ) where
+
+import Data.Text (stripEnd)
+import Hintman.Suggestion.Core (Line(..), LineState(..), Suggestion(..))
+
+
+toLines :: Text -> [Line]
```

Comment:
I guess @sphaso was proposing to have function `toLines` in `Hintman.Suggestion.Core`, but it should only convert `Text` into `[Line]` without changing lines content.
In this module you can have just have a function `FilePath -> [Line] -> [Suggestion]` which will on one bypath check on trailing spaces and write the suggested line.